ubuntudesign/vanilla-brochure-theme
# Add adaptor class to lists to split them

## Done
Added an adaptor class to the list patterns from Vanilla, which splits the list items over two columns.

## QA
- Run `gulp develop`
- Open example/_patterns_list-split.html
- Check that the lists are in two columns
- Modify the demo and try and break it
- Check on small screens that the list stacks in the correct order

## Details
This is a new class `is-split` which only works with list patterns.

Fixes https://github.com/ubuntudesign/vanilla-brochure-theme/issues/9
